subsys/storage/stream/Kconfig

```diff
@@ -17,6 +17,15 @@ config STREAM_FLASH_ERASE
 	  If disabled an external actor must erase the flash area being written
 	  to.
 
+config STREAM_FLASH_SAVE_PROGRESS
+	bool "Store flash write progress"
+	depends on SETTINGS
+	depends on !SETTINGS_NONE  # FIXME: Is this required?
```

Comment:
I guess you need to depend on some backend being chosen, does this handle that case?

Reply:
Yeah, I think so